### ISSUE TITLE
Type hints for aliases

### ISSUE THREAD

**Issue by shyun3** [nneonneo/fixedint#4]
Hello, `pyright` is giving me errors about unknown types when I try importing the aliases such as `UInt8` and `UInt16`. Would it be possible to add proper type hint support for these?

**Comment by nneonneo** [comment]
Proposed fix: add an `aliases.pyi` with the following:

```py
from fixedint.base import FixedInt, MutableFixedInt

class Int8(FixedInt): ...
class Int16(FixedInt): ...
class Int32(FixedInt): ...
class Int64(FixedInt): ...
class UInt8(FixedInt): ...
class UInt16(FixedInt): ...
class UInt32(FixedInt): ...
class UInt64(FixedInt): ...
class MutableInt8(MutableFixedInt): ...
class MutableInt16(MutableFixedInt): ...
class MutableInt32(MutableFixedInt): ...
class MutableInt64(MutableFixedInt): ...
class MutableUInt8(MutableFixedInt): ...
class MutableUInt16(MutableFixedInt): ...
class MutableUInt32(MutableFixedInt): ...
class MutableUInt64(MutableFixedInt): ...
```

Can you try this out and see if it fixes your issue? If so, I'll add it to the repo and publish.

**Comment by shyun3** [comment]
@nneonneo Yes, this worked.

**Comment by shyun3** [comment]
@nneonneo Sorry, just a minute. Now I'm getting errors about no constructor expected when I try to create a value, such as `UInt8(1)`:
```
Expected no arguments to "UInt8" constructor (Pyright reportGeneralTypeIssues)
```

**Comment by nneonneo** [comment]
Thanks. Would you mind trying the [typehints](https://github.com/nneonneo/fixedint/tree/typehints) branch and seeing if that works for you?

**Comment by shyun3** [comment]
I tried installing the branch as follows:
```sh
pip install git+https://github.com/nneonneo/fixedint@typehints
```
and I do receive updated `.py` files as in 418c239, but I don't see any of the added `.pyi` files.

**Comment by nneonneo** [comment]
Oops, I didn't include the .pyi files (sorry, I am new to packaging type hints). Try it now.

**Comment by shyun3** [comment]
Ok, everything looks like it works now. I haven't exercised every functionality, but the constructor seems to pass the type check now.

**Comment by nneonneo** [comment]
I just published [v0.2.0](https://pypi.org/project/fixedint/0.2.0/) which includes the new type hints (and a few little bugfixes). Thanks again for your report.
